scalableminds/webknossos
# Fix styling for login pages for dark mode users

This PR fixes some styling issues for dark mode users after logging out. The default theme for WEBKNOSSOS is light, especially when logged out. This fix, restores light mode on each log out to fix the UI. Theme switching in this case is not persisted as a user choice, though, in order to return to dark mode on the next login for dark mode users.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
1. Log in 
2. Switch to dark mode
3. Log out.
4. Login page should look correct, i.e. have hints for each input field
5. Log in again. You should be in dark mode once more.

### Issues:
- fixes #6885 

------
(Please delete unneeded items, merge only when none are left open)
